<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupOptionsWidget</class>
 <widget class="QWidget" name="StartupOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>On Startup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="noFiles">
        <property name="text">
         <string>Don't open any files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="blankFile">
        <property name="text">
         <string>Open a blank file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="reopenFiles">
        <property name="text">
         <string>Re-open the files that were open last</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="setFiles">
        <property name="text">
         <string>Open the following files (one file per line):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="2" colspan="3">
       <widget class="QPlainTextEdit" name="fileList">
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="currentFiles">
        <property name="text">
         <string>Use Current Files</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="confirmOnExit">
        <property name="text">
         <string>Prompt when closing PonyEdit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
